test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -3004,7 +3004,7 @@ TEST_F(HttpFilterTest, PerRouteCheckSettingsWorks) {
   failure_mode_allow: false
   )EOF");
 
-  // Set allow_partial_message to true and max_request_bytes to 10 on the per-route filter.
+  // Set allow_partial_message to true and max_request_bytes to 5 on the per-route filter.
```

Comment:
Not your change, Could you please also fix the typo in line#2977 while you are already here?

Reply:
Done! Thanks for pointing it out.